nav2_system_tests/src/updown/test_updown_launch.py

```diff
@@ -0,0 +1,147 @@
+# Copyright (c) 2018 Intel Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import os
+
+from ament_index_python.packages import get_package_prefix
+from ament_index_python.packages import get_package_share_directory
+from launch.conditions import IfCondition
+from launch.conditions import UnlessCondition
+
+import launch.actions
+
+
+def generate_launch_description():
+
+    # Configuration parameters for the launch
+
+    world = launch.substitutions.LaunchConfiguration('world')
+    params_file = launch.substitutions.LaunchConfiguration('params',
+            default=[launch.substitutions.ThisLaunchFileDir(), '/nav2_params.yaml'])
+
+    declare_world_cmd = launch.actions.DeclareLaunchArgument(
+            'world',
+            default_value=os.path.join(get_package_share_directory('turtlebot3_gazebo'), 'worlds/turtlebot3_worlds/burger.model'),
```

Comment:
Rather than add a dependency on the turtlebot 3 packages here, I think we just want to keep a local copy of the important files and use them directly.

Turtlebot 3 has created a dependency on our repos. For us to have a dependency on theirs gives us a kind of ugly circular dependency that will probably come back to haunt us in the future.

Reply:
Removed the dependency. Now using the same world as the system test.